client/myapp/src/Components/Pages/myOrders.js

```diff
@@ -26,44 +27,54 @@ function MyOrdersComp(props) {
     if (member) {
       getMemberOrders();
     }
-  })
+  },[])
 
-  const getMemberOrders = () => {
-    let ordersId = member.orders;
 
-    ordersId.forEach(element => {
-      getAllOrders(element);
-    });
-  }
+  const getMemberOrders = async() =>{
+    let resp1 = await utils.getMember(member._id)
```

Comment:
if you already got the memeber and the member_id why to pull it again from the DB?

Reply:
Done